pubpub/pubpub
# update: pub frontend metadata

This PR adds a bunch of metadata to Pub pages to be better conforming for Zotero, Google Scholar, and other services using the [Google Scholar / Highwire](https://scholar.google.com/intl/en/scholar/inclusion.html#indexing) schema. Included:
- A text representation of an abstract, if present
- PDF downloads
- Specific metadata for books, conferences, and issues

_Test plan_
- Create a Zotero account and Install Zotero and the Zotero chrome plugin (https://www.zotero.org/)
- Visit a pub in a book (https://hdsr.duqduq.org/pub/d1z0dryx), conference (https://hdsr.duqduq.org/pub/3mgfrtt1), and journal issue (https://hdsr.duqduq.org/pub/hx3pgxjc) and ensure that Zotero properly identifies their type and pulls in correct metadata fields for the types (book: ISBN, book title, chapter title; conference: conference title; journal issue: ISSN, volume, issue, journal title).
- Visit a pub that has a generated pdf download and ensure Zotero picks that up as the full text PDF (https://hdsr.duqduq.org/pub/hx3pgxjc)
- Visit a pub that has a formatted PDF download and ensure Zotero picks that up as the full text PDF (https://hdsr.duqduq.org/pub/iu26pfw1)
- Visit a pub with a complex abstract (https://hdsr.duqduq.org/pub/hx3pgxjc) with multiple inline things and ensure that it is: adding link URLs to the text where appropriate, and outputting latex math where appropriate. 
- Ensure that pubs with and without abstracts merge properly and can go through review properly.
